server/ping/routes.go

```diff
@@ -0,0 +1,36 @@
+package ping
```

Comment:
I'm not sure I understand the purposes of this being an entire package.

Reply:
`ping` as a package is mostly designed to be a first place someone can look to understand how a package is laid out. Now that the `users` package is pretty much laid out, there probably is not much reason for this to be here anymore.

Reply:
that makes sense, in which case a package comment explaining that would be valuable. I think we're underutilizing package comments in general.

Reply:
Yeah makes sense, will keep that in mind in the future. For now, I've opted to just remove this package.